EnglishTranslation-master/ITEM.tsv

```diff
@@ -10193,46 +10193,46 @@ ITEM_20160224_010192	Rune Stone of Justice
-ITEM_20160310_010222	With every new piece of equipment, the power bestowed by the goddess of the earth is newly imparted on it.
-ITEM_20160310_010223	Users of this equipment may not realize, but designers and artisans know that obtaining the blessing of Goddess Zemyna for using leather is not as easy as it is for using cloth or metal. But thanks to their effort it shines today on the battlefield.
+ITEM_20160310_010219	Those who worship the great goddess will understand humility. This equipment proves that this is true not only for clerics, but for wizards too.
+ITEM_20160310_010220	Whoever carries this will become the master of the earth and receive its grace with every step. However, maintaining this equipment is a problem for humans in regard to cost, and of no concern to the goddess.
+ITEM_20160310_010221	Regardless of what rights the goddess of the earth may give, those who carry and use this equipment do so to their own abilities. Do you not wonder whether it possesses an even greater power?
+ITEM_20160310_010222	With every new piece of equipment, the power bestowed by the goddess of the earth is imparted on it.
+ITEM_20160310_010223	Users of this equipment may not realize, but designers and artisans know that obtaining the blessing of Goddess Zemyna when using leather is not as easy as it is for cloth or metal. Thanks to their effort, it shines today on the battlefield.
 ITEM_20160310_010224	Compared to other leather equipment, it is more readily blessed by Goddess Zemyna since it comes into direct contact with the ground.
 ITEM_20160310_010225	Some Lolopanther equipment was not also made using the power of the owner of the tower, Goddess Zemyna, but also of the invading demons. In this case, it is important to maintain balance for equipment.
-ITEM_20160310_010226	Many of the efforts to take away the power of the demons and what they were targeting remains unknown, buried in the history of the Earth Tower.
-ITEM_20160310_010227	The goddess of the earth is glad to offer humans the ores forged at her hands. However, she does not look kindly on those who carelessly use and discard them.
-ITEM_20160310_010228	The goddess of the earth may have allowed humans to carry weapons; that does not mean She hopes they will bring war and bloodshed. There is a special reason to make use of this particular piece of equipment.
+ITEM_20160310_010226	Many of the efforts to take away the power of the demons and what they were targeting remains unknown.  The answers lay buried in the history of the Earth Tower.
+ITEM_20160310_010227	The goddess of the earth is glad to offer humans the ores forged at her hands. However, she does not look kindly on those who take her gifts for granted.
+ITEM_20160310_010228	The goddess of the earth may have allowed humans to carry weapons; that does not mean she hopes they will bring war and bloodshed. There is a special reason to make use of this particular piece of equipment.
```

Comment:
I don't think "She" should be capitalized when "her" isn't in other situations.  I also think "She" isn't capitalized in other instances.